1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,190 @@
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        And they see a Button "<Persist>" desided by Project's "<Status>"
+        And they see a Next Button
+        Examples:
+            | Status    | Persist |
+            | Published | Publish |
+            | Draft     | Save    |
+
+    @xxx
+    Scenario: Curator opens a Project in the Project Wizard
+        # Same end result as "User Edits a Project in the Project Wizard"
+        # but this one enables a Curator to edit projects where he's not an
+        # participant
+        Given User opens Landing Page for Project
+        And User is Curator on Project's Project Owner's Institution
+        And the project lacks an Approval of type "REK"
+        # A "REK" Approved project is a Health Project. 
+        # See health_related_projects.feature for details.
+        When they click the Edit button
+        Then User sees the Project Wizard
+        And it contains data about the Project
+
+    @xxx
+    Scenario Outline: User opens Participants tab for Project
+        Given User sees the Project Wizard
+        And User has the "<Role>" role in the project
+        When they click the Participants tab
+        Then they see lists of:
+            | Project Managers     |
+            | Project Participants |
+        And they see Buttons:
+            | Add Project Manager     |
+            | Add Project Participant |
+        And they see a Previous Button
+        And they see a Support Button
+        And they see a Button "<Persist>" desided by Project's "<Status>"
+        And they see a Next Button
+        Examples:
+            | Role                  |
+            | Curator               |
+            | Project Owner         |
+            | Project Manager       |
+        Examples:
+            | Status    | Persist |
+            | Published | Publish |
+            | Draft     | Save    |
+
+    @xxx
+    Scenario: Local Project Manager opens Participants tab for Project
+        Given User sees the Project Wizard
+        And User has the Local Project Manager role in the project
+        When they click the Participants tab
+        Then they see lists of:
+            | Project Managers     |
+            | Project Participants |
+        And they see Buttons:
+            | Add Project Participant |
+        And they see a Previous Button
+        And they see a Support Button
+        And they see a Button "<Persist>" desided by Project's "<Status>"
+        And they see a Next Button
+        Examples:
+            | Status    | Persist |
+            | Published | Publish |
+            | Draft     | Save    |
```

Comment:
Mye duplisering på de to scenarioene over her (like sjekker på begge). Kan ikke denne (`Local Project Manager opens Participants tab for Project`) endres til `User opens Participants tab for Project`, men ellers se lik ut, så kan vi erstatte `User opens Participants tab for Project` med:
```
Scenario Outline: Privileged user opens Participants tab for Project
    Given User opens Participants tab for Project
    When the User have Role "<Role>"
    Then they see Add Project Manager Button
    Examples:
        | Role                  |
        | Curator               |
        | Project Owner         |
        | Project Manager       |
```

Reply:
Takk. Done.